language/tools/move-cli/src/base/disassemble.rs

```diff
@@ -21,6 +21,9 @@ pub struct Disassemble {
     /// The name of the module or script in the package to disassemble
     #[clap(long = "name")]
     pub module_or_script_name: String,
+    /// Also print the raw disassembly using Rust's Debug output, at the end.
+    #[clap(long = "Xdebug")]
+    pub debug: bool,
```

Comment:
This is not the main change, but it adds a flag to the disassembler (`--Xdebug`) that adds the raw `Debug` output of the `CompiledModule` or `CompiledScript` to the end of the usual non-interactive output.

Ideally, I would have added this to the transactional-test commands, but `move-cli/tests/build_tests` doesn't use this, and these tests are ideally placed there, because we need to use named addresses with numerical assignments.

Reply:
What's the reasoning behind the name `Xdebug`?

Reply:
I'm used to the `-X` flags on compilers being the hidden ones that you're not really supposed to use as a user of the compiler (i.e. they are more intended for the compiler authors) -- I think it's a JVM convention?

They are often also hidden from the help/usage output as well, but I couldn't get that to work with `clap_derive`.

I don't have a strong preference on it though -- if we're alright with this being here, we can drop the "X".

Reply:
Ah, that's a neat convention. I'm fine with it, we can always change it too. 
But maybe @tzakian has some thoughts?